tests/src/JIT/Directed/nullabletypes/Desktop/StructDefinitions.cs

```diff
@@ -236,6 +236,11 @@ static Helper()
         GCHANDLE = GCHandle.Alloc(Console.Out);
     }
 
+    public static void Shutdown()
```

Comment:
It would be better for this `Helper` to create object with finalizer to clean this up or register unload event to clean this up, rather than changing hundreds of tests to call the Shutdown method manually.